### PR TITLE
[RFR] [Renderer] removed outdated helpers and updated Renderer default parameters #391 #423

### PR DESCRIPTION
**Pull request by eric-chau**
More details in #391 and #423.

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
:+1:  (and sorry for StyleCi, I have removed the service).

**Comment by mickaelandrieu** [comment]
@eric-chau if you consider your pull request done, you can merge.

**Comment by eric-chau** [comment]
Yeah but it's not yet. I will ping you one last time when it is over to let you review this ;)

**Comment by eric-chau** [comment]
This PR is finish, I cleaned every unused helpers and added some default parameters: ``app``, ``bbtoken``, ``request`` and ``routing``.

File: Renderer/Helper/isGranted.php [hunk 1]
```diff
@@ -1,54 +0,0 @@
-<?php
```
Comment (eric-chau) [inline Renderer/Helper/isGranted.php]: @crouillon, are you ok with the deletion of this helper too? I checked every templates and it is not used.

File: Renderer/Renderer.php [hunk 2]
```diff
@@ -1041,6 +1050,28 @@ private function renderTemplate($isPartial = false, $isLayout = false)
...
+            array_merge($this->getAssignedVars(), $this->getBBVariable())
+        );
+    }
+
+    /**
+     * Returns default parameters that are availables in every templates.
+     *
+     * @return array
+     */
+    private function getBBVariable()
```
Comment (eric-chau) [inline Renderer/Renderer.php]: @crouillon, I finally added a ``bb`` variable to every templates that gives access to ``token``, ``request``, ``debug`` and ``routing``. You confirm that you're still okay with this?

File: Renderer/Helper/isGranted.php (hunk 1 above)
Comment (crouillon) [inline Renderer/Helper/isGranted.php]: ok

File: Renderer/Renderer.php (hunk 2 above)
Comment (crouillon) [inline Renderer/Renderer.php]: Yes i aggree :+1:

**Comment by mickaelandrieu** [comment]
For me it's good!  Nice job @eric-chau

**Comment by mickaelandrieu** [comment]
aww sorry you may forget to remove '' app'' helper or did you change your mind?  This helper should be used wisely lol

**Comment by eric-chau** [comment]
Sure you're right!

**Comment by eric-chau** [comment]
oh well I really didn't know what happened but there are many changes I've done that are not saved. I will change it now :s

**Comment by eric-chau** [comment]
Fixed, sorry for the fail.

**Pull request merged**